### ISSUE TITLE
pyperclip.paste() returns ''

### ISSUE THREAD

**Issue by Gizmo11** [asweigart/pyperclip#114]
I'm new to Python and I've looked at other issues but I can't seem to find one that is doing the same thing as mine.
I am running Windows 10 and Python 3.5.3 and I have installed pyperclip 1.6.0.
I can import pyperclip without any errors in the command window or in IDLE.
When I copy something to the clipboard I can paste it using ctrl V but when I type:
`pyperclip.paste()`
I get
`''`
even though I know there is text on the clipboard.  I get the same in both IDLE and the command window.
also typing
`pyperclip.copy('Hello World!')`
doesn't return an error, but I get an empty clipboard after that.

No idea what I'm doing wrong.

**Comment by aydindemircioglu** [comment]
yes, same problem here, but only with a html source code and only if the text exceeds magically around 261000 chars. works with other text even if its a million chars..

**Comment by simonebenati** [comment]
Same issue, still on going as of april 2022

**Comment by alcpereira** [comment]
Same on MacOS. I can't find a consistent scenario where it happens, but I'd say it happens less than 10% of the time.

I suspect it comes from .copy() itself as when I'm manually pasting after the script execution the clipboard is still empty.

I also use Alfred on Mac that shows the clipboard history but they don't display those empty lines (and it makes sense).
